### PR TITLE
fix exception message on rcl_clock_init.

### PR DESCRIPTION
**Pull request by fujitatomoya**
Signed-off-by: Tomoya.Fujita <Tomoya.Fujita@sony.com>

### CODE REVIEW

**Comment by fujitatomoya** [comment]
@ivanpauno 

only message fix, could you help to merge?

**Comment by fujitatomoya** [comment]
@ivanpauno 

friendly ping,

File: rclcpp/src/rclcpp/clock.cpp [hunk 1]
```diff
@@ -32,7 +32,7 @@ class Clock::Impl
   {
     rcl_ret_t ret = rcl_clock_init(clock_type, &rcl_clock_, &allocator_);
     if (ret != RCL_RET_OK) {
-      exceptions::throw_from_rcl_error(ret, "could not get current time stamp");
+      exceptions::throw_from_rcl_error(ret, "Failed to initialize rcl clock");
```
Comment (ivanpauno) [inline rclcpp/src/rclcpp/clock.cpp:35]: we usually start error messages in lower case:
```suggestion
      exceptions::throw_from_rcl_error(ret, "failed to initialize rcl clock");
```

Comment (fujitatomoya) [reply rclcpp/src/rclcpp/clock.cpp:35]: got it! take your suggestion fix.

**Comment by fujitatomoya** [comment]
@ivanpauno 

done!

**Comment by ivanpauno** [comment]
@fujitatomoya can you contribute a backport to Foxy?
If the change applies to dashing/eloquent branches, it would be good to have those backports too

**Comment by fujitatomoya** [comment]
@ivanpauno 

> can you contribute a backport to Foxy?
> If the change applies to dashing/eloquent branches, it would be good to have those backports too

I will check and do so.

**Comment by fujitatomoya** [comment]
@ivanpauno 

could you help to merge the following backports?

- dashing: https://github.com/ros2/rclcpp/pull/1195
- eloquent: https://github.com/ros2/rclcpp/pull/1194
- foxy: https://github.com/ros2/rclcpp/pull/1193

**Comment by ivanpauno** [comment]
@fujitatomoya I've added as reviewers the respective rosbosses of those distros.

**Pull request merged**